src/make-hid.sh

```diff
@@ -1,25 +1,74 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
If you change this to `#!/usr/bin/env bash` then it works cross-platform. Nix (and others) don't have /bin/bash

Reply:
I forgot about that one. I'll amend that.

Reply:
Fixed in commit 74a2406d2b88448ad483194856365e5755c0f6e4